javascript/first-js-oop-project.md

```diff
@@ -92,6 +108,11 @@ This mini project will help you to learn basic concepts about objects in JavaScr
    // in order to check whether the elements are pushed, use console.log
    console.log(event_array);
    ```
+1. Write a function named `matchTickets` that will take in two values for a price range, and return a list of ticket types available. The method should look like this when it is called:
```

Comment:
maybe the function should be called `searchTickets`? And clarify what parameters the function takes in.